site/src/routes/docs.svelte

```diff
@@ -20,6 +20,7 @@
 			'docs#use-it',
 			[
 				['mdsvex', 'docs#mdsvex-1', true],
+				['mdsvex with sapper', 'docs#mdsvex-with-sapper', true],
```

Comment:
Like above, we can move this to the the `integrations` section:

Change the commented out line 50 to be:

```js
[
  "Integrations", 
  "docs#integrations"
  [
    ['sapper', 'docs#sapper', false],
  ],
],
```

I think this will add it to the navigation correctly.